packages/wrangler/src/index.ts

```diff
@@ -753,38 +774,6 @@ export function createCLIParser(argv: string[]) {
 	]);
 	registry.registerNamespace("queues");
 
-	registry.define([
```

Comment:
Just to verify my understanding, the commands we are deleting are:
- in open beta
- undocumented
- and hidden in wrangler's help menu
correct?

Even if so I want to point out that removing them might cause breakage for users. It might be a very small subset of users, but it is a possibility. Ultimately, this is a product decision, and if product is OK with this, we won't push back against it, but it felt the right thing to flag. I'm also unsure if this is just a command rename situation or if the new commands have a diff behaviour.

A possibility to avoid the potential breakage, is to alias the old commands, not sure if this was considered. Again, up to the Queues team, but wanted to provide options ♥️